test/hotspot/jtreg/compiler/loopopts/parallel_iv/TestParallelIvInIntCountedLoop.java

```diff
@@ -345,8 +346,8 @@ private static void testCorrectness() {
 
             // also test with random init and init2
             int init1 = rng.nextInt();
-            int init2 = rng.nextInt(Integer.MIN_VALUE + i + 1, i);
-            long init1L = rng.nextLong(Long.MIN_VALUE + i + 1, i);
+            int init2 = rng.nextInt(Integer.MIN_VALUE + i + 1, i); // Similarly, avoid (i - init2) from overflowing.
```

Comment:
What does the "similarly" refer to? To comment line 347 or line 321? It's unclear to me. I would remove it.

Reply:
Was refering to line 321. Updated comments.